### ISSUE TITLE
slow update with obs 25.0.8 on Linux

### ISSUE THREAD

**Issue by mattvenn** [univrsal/spectralizer#35]
I'm using the Latest release  with ubuntu 18.

It seems to work ok with a webcam providing audio, but with either the laptop mic or an external usb soundcard as the audio device the fft will only update once a second or so.

Changing sample frequency from 44.1 to 48 or back again sometimes makes it work for a time and then the next time I start obs it will be stuck again.

Any ideas on how to debug this?

**Comment by univrsal** [comment]
> Any ideas on how to debug this?

Nope, not really. I have had a few people with issues, but I can never reproduce them and I know next to nothing about digital signal processing.

**Comment by ThomasHineXYZ** [comment]
I'm also having this issue, with the same result as mattvenn. 

When "Desktop Audio" is selected, it is slow, but when I choose my microphone, my capture card audio, or any other option it updates just fine. 

I'm on OBS 26.0.2-3 if that helps at all...
